compiler/rustc_typeck/src/check/intrinsic.rs

```diff
@@ -367,6 +367,14 @@ pub fn check_intrinsic_type(tcx: TyCtxt<'_>, it: &hir::ForeignItem<'_>) {
 
             sym::nontemporal_store => (1, vec![tcx.mk_mut_ptr(param(0)), param(0)], tcx.mk_unit()),
 
+            sym::raw_eq => {
+                let param_count = if intrinsic_name == sym::raw_eq { 2 } else { 1 };
```

Comment:
This is matching on the intrinsic name already?

Reply:
Oops!  Good catch.  (Remnant of a previous attempt which included a second intrinsic.)

Reply:
still there, did you forget to push?

Reply:
It's fixed -- https://github.com/rust-lang/rust/pull/85828/files#diff-b2a7c31c785c36fc43ebd3ba40a8a1571af0cf6fb3852594055305ac5bc58e88R371 -- GitHub just isn't showing it on the conversation page.